### PR TITLE
5.2.2 component information about mentor change password

### PR DESCRIPTION
**Pull request by staspitsyk**
Added change password modal

### CODE REVIEW

File: front/src/pages/teachers/teachers-information/index.js [hunk 1]
```diff
@@ -65,7 +80,7 @@ class TeachersInformation extends Component {
                 <Groups>{`${groupsAmount} groups`}</Groups>
                 <Students>{`${studentsAmount} students`}</Students>
               </p>
-              <Button type="primary">Change password</Button>
+              <Button data-id={id} onClick={this.showModalChangePassword} type="primary">Change password</Button>
```
Comment (Alan18081) [inline front/src/pages/teachers/teachers-information/index.js]: To pass id you can define `onClick` in the format `() => this.showModalChangePassword(id)`, it is much easier to understand

File: front/src/containers/modal-change-password/index.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ class ChangePassword extends Component {
 
     render() {
         const { errorMessage, visible, handleCancel, user } = this.props;
-
+        console.log(this.props);
```
Comment (Alan18081) [inline front/src/containers/modal-change-password/index.js]: Useless `console.log`

Comment (staspitsyk) [reply front/src/containers/modal-change-password/index.js]: Removed

File: front/src/pages/teachers/teachers-information/index.js (hunk 1 above)
Comment (staspitsyk) [reply front/src/pages/teachers/teachers-information/index.js]: Done

File: front/src/pages/teachers/teachers-information/index.js [hunk 3]
```diff
@@ -65,7 +80,7 @@ class TeachersInformation extends Component {
                 <Groups>{`${groupsAmount} groups`}</Groups>
                 <Students>{`${studentsAmount} students`}</Students>
               </p>
-              <Button type="primary">Change password</Button>
+              <Button data-id={id} onClick={() => this.showModalChangePassword(id)} type="primary">Change password</Button>
```
Comment (Alan18081) [inline front/src/pages/teachers/teachers-information/index.js]: You don't need `data-id` anymore)

Comment (staspitsyk) [reply front/src/pages/teachers/teachers-information/index.js]: Removed

**Review by Alan18081 — APPROVED** [review]

**Pull request merged**